swiggy-pdp-cart/src/components/Main/Content/helper/helper.js

```diff
@@ -0,0 +1,9 @@
+export function categoryNameById(id, list) {
```

Comment:
Having a file called helper inside helper does not make sense neither offer any benefit.

Rename the folder to helpers as it can house more than one helper down the line.

Rename the helper file to <module.context>.js format, so here it could be `content.heading.js` or `content.general.js`